test/specs/annotation.tests.ts

```diff
@@ -78,7 +78,7 @@ describe('Annotate unary lookup', () => {
 describe('Path expression test', () => {
 	it('Path expression test', () => {
 		// The same query also triggers the path expression
-		assertValueType('map{"num":1}[?num]', ValueType.ITEM, undefined);
+		assertValueType('map{"num":1}[?num]', ValueType.MAP, undefined);
```

Comment:
We want more tests to test if the behavior you're trying to achieve is correct!

Reply:
Most of this code is tested via other tests but I can add some specific ones